src/StateSubject.js

```diff
@@ -0,0 +1,61 @@
+import { Subject } from 'rxjs/Subject';
+
+// Used as a placeholder value so we can tell
+// whether or not the real state has been set,
+// even if the real state was `undefined`
+const UNSET_VALUE = {};
+
+// We can't use BehaviorSubject because when we create
+// and give state$ to your epics your reducers have not
+// yet run, so there is no initial state yet until the
+// `@@redux/INIT` action is emitted. BehaviorSubject
+// requires an initial value (which would be undefined)
+// and if an epic subscribes to it on app boot it would
+// synchronously emit that undefined value incorrectly.
+export class StateSubject extends Subject {
+  constructor(store) {
+    super();
+    // If you're reading this, keep in mind that this is
+    // NOT part of the public API and will be removed!
```

Comment:
Could always hide it with a local symbol.

```
// Way above 
const ACTUAL_STORE = Symbol('Actual Store');

  this[ACTUAL_STORE] = store;
  this[ACTUAL_VALUE] = UNSET_VALUE;
```

I do of course realize you can get at this property still using `Object.getOwnPropertySymbols`. It's a _little_ more hidden though without having to make a closure around a class.

Reply:
Trying to avoid Symbols since they aren't in IE 11 and some older versions of the others..but tbh I don't even know if redux-observable works in IE 11. I may use others features that break stuff.